OrbitGl/CaptureWindow.cpp

```diff
@@ -629,10 +629,16 @@ void CaptureWindow::ToggleDrawHelp() {
   NeedsRedraw();
 }
 
-Batcher& CaptureWindow::GetBatcherById(uint32_t batcher_id) {
-  return batcher_id == PickingID::BatcherId::kTimeGraph
-             ? time_graph_.GetBatcher()
-             : ui_batcher_;
+Batcher& CaptureWindow::GetBatcherById(BatcherId batcher_id) {
+  switch (batcher_id) {
+    case BatcherId::kTimeGraph:
+      return time_graph_.GetBatcher();
+    case BatcherId::kUi:
+      return ui_batcher_;
+    default:
+      CHECK(false);
+      return ui_batcher_;
+  }
```

Comment:
you can add UNREACHABLE() after the switch just in case.

Reply:
Done